packages/firebase_auth/firebase_auth_platform_interface/pubspec.yaml

```diff
@@ -12,16 +12,16 @@ environment:
 
 dependencies:
   _flutterfire_internals: ^1.3.4
-  collection: ^1.16.0
+  collection: ^1.17.1
   firebase_core: ^2.15.0
   flutter:
     sdk: flutter
-  meta: ^1.8.0
-  plugin_platform_interface: ^2.1.3
+  meta: ^1.9.1
+  plugin_platform_interface: ^2.1.4
 
 dev_dependencies:
   firebase_core_platform_interface: ^4.8.0
   flutter_test:
     sdk: flutter
-  mockito: ^5.4.0
-  pigeon: ^9.2.5
+  mockito: ^5.4.2
+  pigeon: ^10.1.3
```

Comment:
@Lyokone - wondering if this is going to cause an issue?

Reply:
Should probably be removed 👍 Or we would need to regenerate Pigeon files, but it's probably better to do it in another PR